### ISSUE TITLE
FreeBSD stuck on Version 20.00

### ISSUE THREAD

**Issue by anteract** [webmin/authentic-theme#1627]
Upgrading either with the github tool, or the webmin upgrade tool, Webmin on FreeBSD 13 won't go past 20.00.  Says its installing, but happily returns right back where it started.  The other software is current and updated except for that.

I did get it installed on one server by removing the theme completely and reinstalling it, but that seems wrong.

**Comment by iliajie** [comment]
> Upgrading either with the github tool, or the webmin upgrade tool, Webmin on FreeBSD 13 won't go past 20.00.

Could you provide a screenshot of a page where you're performing upgrade and an error message?

**Comment by anteract** [comment]
Before and after the update, this is the screen.
The info file at:  /usr/local/lib/webmin-2.000/authentic-theme/theme.info

Shows "version=20.02" so perhaps this is nothing but a visual issue.

![update screen](https://user-images.githubusercontent.com/1554315/195677106-eba550cc-170c-4a65-8b85-8058737626cb.jpg)

BUT!  This screen shows update needed every time I load the screen:

![Main Menu Webmin](https://user-images.githubusercontent.com/1554315/195678018-cb94c04e-81d9-404f-83b4-e262a6efdfce.jpg)

Apparently on digging, its an annoyance more than a broken thing.  other servers (centos) don't do it, so I tend to notice it more

**Comment by iliajie** [comment]
What is the actual Webmin version that you're using? Is `bash` package installed on that FreeBSD system?

Theme update script uses a directory relative to where it's run from, so the update should happen disregard to the directory Webmin is installed to.

If you run `./theme-update.sh` script from a console -- does it fail with any error?

Finally, you can always disable theme updates checks.

**Comment by anteract** [comment]
Webmin 2.000
Bash version 5.2.2(1)-release

And I figured it out.

Webmin on FreeBSD 's package manager gets updated infrequently, so I often just use Webmin's update now feature, which ends up with multiple versions of webmin installed in the "lib" folder.  .../etc/webmin/start was starting the one in /usr/local/lib/webmin, but the theme updater (on the Dashboard) was updating /usr/local/lib/webmin-2.000.  I stopped Webmin, manually cleared the extra folders (there were many) and the update worked fine, now says 20.02-2.  Probably never see this again, and when the next Webmin update came along it would have fixed it too.  And yes, I had tried restarting Webmin a few times (lots) before ever bringing this up.

So, webmin updates always updated by webmin and this would have been fine, but updating with the pkg tool, authentic didn't get the memo.  (all other features of webmin were fine of course)
